Mathlib/Order/Chain.lean

```diff
@@ -6,6 +6,7 @@ Authors: Johannes Hölzl
 import Mathlib.Data.Set.Pairwise.Basic
 import Mathlib.Data.Set.Lattice
 import Mathlib.Data.SetLike.Basic
+import Mathlib.Order.Sublattice
```

Comment:
This is *far* too heavy an import. Just create a new file if there isn't a better place.